flow360/component/simulation/models/surface_models.py

```diff
@@ -69,7 +69,7 @@ class Temperature(SingleAttributeModel):
 
     type_name: Literal["Temperature"] = pd.Field("Temperature", frozen=True)
     # pylint: disable=no-member
-    value: Union[TemperatureType.Positive, StringExpression] = pd.Field(
+    value: Union[StringExpression, TemperatureType.Positive] = pd.Field(
```

Comment:
Did you change all occurrences?

Reply:
Yes, just double checked again and have changed all occurrences.